blog/_posts/2017-07-06-introducing-team-clojurians.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Introducing Team Clojurians
+layout: post
+created_at: Thu July 6 2017
+permalink: blog/introduction
```

Comment:
could you change this permalink to `blog/2017-07-06-introducing-team-clojurians`? thank you!